actioncable/lib/action_cable/subscription_adapter/postgresql.rb

```diff
@@ -12,16 +12,16 @@ def initialize(*)
 
       def broadcast(channel, payload)
         with_connection do |pg_conn|
-          pg_conn.exec("NOTIFY #{pg_conn.escape_identifier(channel)}, '#{pg_conn.escape_string(payload)}'")
+          pg_conn.exec("NOTIFY #{pg_conn.escape_identifier(channel.to_s)}, '#{pg_conn.escape_string(payload)}'")
```

Comment:
Ugh, I forget that `String()` falls back to `#to_s`.

In this case, it seems desirable to at least give the option of being string like (via `#to_str`) here and then fall back to `#to_s` in the worst case.